lib/OrdinaryDiffEqRKC/src/OrdinaryDiffEqRKC.jl

```diff
@@ -0,0 +1,53 @@
+module OrdinaryDiffEqRKC
```

Comment:
Let's call it OrdinaryDiffEqStabilizedRK, since technically not all are Chebyshev derived. So the more general term for this class is Stabilized Runge-Kutta methods.

Reply:
Ok. I will do that.